python/lib/dcoscli/tests/integrations/test_marathon_groups.py

```diff
@@ -35,6 +35,11 @@ def test_add_existing_group():
                            stdin=fd)
 
 
+def test_add_non_root_group_from_stdin():
+    _add_group_by_stdin('tests/data/marathon/groups/simple.json')
+    remove_group('/x')
```

Comment:
* What does this test? I see that a group is added and then removed but what are the constraints here?
* How does this test relate to the code changed above?

Reply:
I added a comment

Reply:
I don't think this setup is ideal. What happens if someone changes the `simple.json` but not this line? Since `simple.json` is only a single line I would just pass the literal data here so it's all in one place.

Reply:
I agree but all our tests are using files and all helper functions are relying on file. If we would like to keep everything in this test I'd need to create temp file or create new helper that can handle string command stdin.

See:
* https://github.com/dcos/dcos-core-cli/blob/cdbbfce1579bf0c4d706bfb3675ec775d0f0713a/python/lib/dcoscli/tests/integrations/test_marathon_groups.py#L120
* https://github.com/janisz/dcos-core-cli/blob/55b3085ed15a1b26df8f0fbce58c15958c2541cc/python/lib/dcoscli/dcoscli/test/common.py#L17-L24